lib/servo.js

```diff
@@ -275,8 +274,8 @@ Servo.prototype.to = function(degrees, time, rate) {
   if (this.invert) {
     degrees = Board.map(
       degrees,
-      this.range[0], this.range[1],
-      this.range[1], this.range[0]
+      0, 180,
+      180, 0
```

Comment:
I think I know what my disconnect is: I've assumed that a user specified range was exclusively to define the physical range. I've used range: [a, b] to limit the bounds of servos that have 180° range